docs/source/conf.py

```diff
@@ -374,13 +372,15 @@
 
 # Example configuration for intersphinx: refer to the Python standard library.
 intersphinx_mapping = {'python': ('http://docs.python.org/', None),
```

Comment:
intersphinx is a really interesting solution to the problems we brought up really early on in building the docs. Nice find.